ktpbase/config/config.py

```diff
@@ -0,0 +1,2 @@
+class ConfigManager:
```

Comment:
Maybe add a comment to these files that they are nescesarry at the moment for unit tests.

Reply:
Good point. This should be solved by #a1978c5c487b6f603f9b0b8916bcb4f7e57e0905